azure-pipelines.yml

```diff
@@ -0,0 +1,150 @@
+trigger:
+- release/*
+  
+pool:
+  vmImage: windows-latest
+
+parameters:
+- name: doPublish
+  default: false
+  type: boolean
+- name: toDeploy
```

Comment:
This isn't required - you can simply deselect the deploy stage if you don't want to deploy

Reply:
Removed in latest commit